gen/builders/state_zero.go

```diff
@@ -126,6 +128,14 @@ func (b *Builder) initializeZeroState() {
 		}},
 	})
 
+	// Chaos asctor.
+	actors = append(actors, ActorState{
```

Comment:
Shouldn't we conditionally append based on the selector? I assume if that happens then SDK's could _not_ implement the actor and still be able to run the other tests? Adding the actor here will change the precondition CIDs - right?

Reply:
Yup! You're absolutely right.